tests/npc_test.cpp

```diff
@@ -0,0 +1,79 @@
+#include "catch/catch.hpp"
+
+#include "player.h"
+#include "npc.h"
+#include "game.h"
+#include "map.h"
+
+#include <string>
+
+void on_load_test( npc &who, calendar from, calendar to )
+{
+    calendar::turn = from;
+    who.on_unload();
+    calendar::turn = to;
+    who.on_load();
+}
+
+void sane( const npc &who )
+{
+    CHECK( who.get_hunger() >= 0 );
+    CHECK( who.thirst >= 0 );
+    CHECK( who.fatigue >= -25 );
+}
+
+TEST_CASE("NPC on_load correctly approximates hunger/thirst/fatigue")
```

Comment:
The main thing these test case names are used for is running just this test, like ```tests/cata_test "NPC on_load correctly approximates hunger/thirst/fatigue"```, so it's nicer to have them be slightly shorter and cli-friendly, like "npc-needs-update-on-load".
Seperate issue, if the test doesn't pass, you can make it not run by default by passing a second argument to TEST_CASE() which is a string starting with a period, including just a period.